core/src/client/lwm2m_static.c

```diff
@@ -599,4 +599,53 @@ AwaError AwaStaticClient_RegisterResourceWithHandler(AwaStaticClient * client, c
                                             NULL, 0 , 0);
 }
 
+AwaError AwaStaticClient_ResourceChanged(AwaStaticClient * client, AwaObjectID objectID, AwaObjectInstanceID objectInstanceID, AwaResourceID resourceID)
+{
+    AwaError result = AwaError_Unspecified;
+
+    if (client != NULL)
+    {
+        ResourceDefinition * definition = Definition_LookupResourceDefinition(Lwm2mCore_GetDefinitions(client->Context), objectID, resourceID);
+        if (definition != NULL)
+        {
+            if (definition->Handlers.Read == NULL)
```

Comment:
Sean and I agree - best not to use a deprecated mechanism in new code. Check is not required.